conf/config.level0.neon

```diff
@@ -55,6 +55,7 @@ rules:
 	- PHPStan\Rules\Properties\AccessStaticPropertiesInAssignRule
 	- PHPStan\Rules\Properties\PropertyAttributesRule
 	- PHPStan\Rules\Variables\UnsetRule
+	- PHPStan\Rules\DateTimeInstantiationRule
```

Comment:
1) Needs to be on level 5 to be consistent with other rules
2) Needs to be only in bleeding edge. See an example of how to set it up: https://github.com/phpstan/phpstan-src/blob/41ac2127687bca4aab6ca0a953860f9de0b3f098/conf/config.level4.neon#L20-L30